TUM Campus App/CardViewController.swift

```diff
@@ -128,3 +150,18 @@ extension CardViewController {
     
 }
 
+
+extension CardViewController: UISearchResultsUpdating {
+    
+    func updateSearchResults(for searchController: UISearchController) {
```

Comment:
I think the search results controller should handle this.
And then the cards view controller is less clutered

Reply:
As I already said, doing it in the ResultsVC makes the search laggy.

Reply:
Ok I'll check it out

Reply:
I think so too